### ISSUE TITLE
Choosing KDE desktop environment in the Everything iso is pretty minimal

### ISSUE THREAD

**Issue by Wojtek242** [Zer0CoolX/Fedora-KDE-Minimal-Install-Guide#1]
I just installed Fedora using the Everything install image and I chose the KDE desktop environment on the list of software when installing and the environment I ended up was pretty minimal. The only "bloatware" were konqueror, kmail, korganiser and the likes. It doesn't even come with okular!

I would recommend giving that a go next time to see how your experience compares with your minimal setup.

**Comment by Zer0CoolX** [comment]
I am installing both in VM's right now using the Everything ISO as of 4/6/2019 in the interest of comparing the results. Mainly interested in how many packages are installed total between them and then maybe more narrowly what KDE apps are installed.

I will post back with results. Thanks.

**Comment by Zer0CoolX** [comment]
So after doing it here is the comparison (exact results may vary person to person):

Using my method with just the required software. **Total packages installed = 1172**
Using the KDE option from the Everything ISO (and no selections made from the right hand column of the sub packages in Anaconda). **Total packages installed = 1575**

**Difference is = 403 additional packages** or an additional ~34% packages installed.

KDE specific packages installed using KDE option from Everything ISO **NOT** installed using my method:

1. Konqueror
2. Kontact
3. Discover (package mgr)
4. dnfdragora (also a package mgr)
5. kcolor chooser
6. kruler
7. gwenview
8. akregator
9. kmail
10. sieve editor
11. contact theme editor
12. KOrganizer
13. KTnef
14. KmouseTool
15. KCharSelect
16. Kgpg
17. KFind
18. Kmail import wizard
19. KCalc
20. KMag
21. Spectacle
22. Kmouth
23. Kwrite

Some non-KDE apps installed that are not with my method:

1. QT5 d-bus debugger (because we all need this handy)
2. Fedora release notes (and this too)
3. Cangjie preferences (input method...wtf is this lol)
4. Disks (if you want it, you could install it...or uninstall it I guess. Or you can just use KDE partition manager thats also installed)

Its possible I missed some packages like dependencies. This is just what I eye balled from the Application launcher.

So in summary, Using the Everything ISO and installing the KDE option is likely better than the KDE spin, but not by much. Dont need 2 package manager GUI's, Dont need 2 disk partition GUIs, etc.

People can make up their minds, and options are good, but unfortunately using the KDE spin or the KDE option results in a lot of excess packages most people do not want nor will they keep. I am going to close this issue.

**Comment by KAMiKAZOW** [comment]
>KDE specific packages installed using KDE option from Everything ISO NOT installed using my method:
> […]
> 5. kcolor chooser

Yes, you do install that.
